### PR TITLE
Update devbox_setup.md

### LINKED ISSUE #777: devbox_setup instructions have broken link for installing PS on Linux
- **OS, version, SKU and CPU architecture used:** None, this issue is in .md file.
- **Application's .NET Target Framework :** N/A 
- **Device:** N/A 
- **SDK version used:** [master@{2019-01-29}](https://github.com/Azure/azure-iot-sdk-csharp/commit/d4d2fd72b25a0ed8280585cfd584a785995c1663) 

## Description of the issue:

Link https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/devbox_setup.md#linuxosx-tested-on-ubuntu-1604 is broken in https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/devbox_setup.md

## Code sample exhibiting the issue:

N/A

## Console log of the issue:

N/A

### PR DESCRIPTION
**Pull request by CIPop**
## Checklist
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [X] This pull-request is submitted against the `master` branch.

## Description of the changes
Fix Linux links.

## Reference/Link to the issue solved with this PR (if any)

Fixes #777

### CODE REVIEW

File: doc/devbox_setup.md [hunk 1]
```diff
@@ -113,4 +113,4 @@ Run the following as admin: `iothub\service\tests\prerequisites\windows_install.
 [UWP]:https://msdn.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide
 [.NET]:https://www.microsoft.com/net
 [UWP]:https://msdn.microsoft.com/en-us/windows/uwp/winrt-components/index
-[Xamarin]:https://www.xamarin.com/
\ No newline at end of file
+[Xamarin]:https://www.xamarin.com/
```
Comment (prmathur-microsoft) [inline doc/devbox_setup.md]: Why do we add link to Xamarin?

Comment (CIPop) [reply doc/devbox_setup.md]: Nothing is actually changed there: not sure why GH considers that a change (maybe spacing?).

**Comment by CIPop** [comment]
Unrelated failures: this is a doc-only change.

**Pull request merged**